dspace-swordv2/src/main/java/org/dspace/sword2/WorkflowTools.java

```diff
@@ -144,38 +171,36 @@ public void startWorkflow(Context context, Item item)
             WorkspaceItem wsi = this.getWorkspaceItem(context, item);
 
             // kick off the workflow
-            boolean notify = ConfigurationManager.getBooleanProperty("swordv2-server", "workflow.notify");
-            if (ConfigurationManager.getProperty("workflow", "workflow.framework").equals("xmlworkflow")) {
-                if (notify) {
-                    XmlWorkflowServiceImpl.start(context, wsi);
-                } else {
-                    XmlWorkflowServiceImpl.startWithoutNotify(context, wsi);
+            boolean notify = ConfigurationManager
+                    .getBooleanProperty("swordv2-server", "workflow.notify");
+            if (ConfigurationManager
```

Comment:
Another case where the if this workflow then that can be replaced by WorkflowServiceFactory.getInstance().getWorkflowService().start(), WorkflowServiceFactory.getInstance().getWorkflowService().startWithoutNotify()